src/assets/wise5/common/user/user.ts

```diff
@@ -0,0 +1,8 @@
+export function usernameComparator(object1: { username: string }, object2: { username: string }) {
+  if (object1.username == null || object2.username == null) {
+    return 0;
+  }
+  const username1 = object1.username.toLowerCase();
+  const username2 = object2.username.toLowerCase();
+  return username1.localeCompare(username2);
```

Comment:
You can use a ternary operator here to remove a return and reduce complexity by 1.
```suggestion
  const username1 = object1.username;
  const username2 = object2.username;
  return username1 == null || username2 == null
    ? 0
    : username1.toLowerCase().localeCompare(username2.toLowerCase());
```